src/index.dev.js

```diff
@@ -4,13 +4,24 @@ require("@babel/register")({
   extensions: [".ts", ".js", ".tsx", ".jsx"],
   plugins: ["babel-plugin-dynamic-import-node"],
 })
-const path = require("path")
+
+// Workaround until more appropriate methods for generating an individual config
+// are implemented.
+process.env.AUTO_CONFIGURE = true
```

Comment:
The `AUTO_CONFIGURE` name is abstract to me, I wonder if there's something that can speak more specifically to what the resulting config accomplishes or avoids?

Reply:
I used the terminology from autotools (automake, autoconf), this flag automatically configures our WebPack configuration so that we can programmatically select a configuration without needing to swap in and out the legacy env variables.